hw/xbox/nv2a/nv2a.h

```diff
@@ -440,9 +440,42 @@ typedef struct NV2ABlockInfo {
 extern const struct NV2ABlockInfo blocktable[];
 extern const int blocktable_len;
 
+DMAObject nv_dma_load(NV2AState *d, hwaddr dma_obj_address);
+void *nv_dma_map(NV2AState *d, hwaddr dma_obj_address, hwaddr *len);
```

Comment:
Why would this need to be public?
Are there any uses outside nv2a.c? Should there be?

*Possibly other similar instances too.*

Reply:
Yes, see nv2a_[pgraph|pfifo].c